app/encryption.py

```diff
@@ -42,22 +18,22 @@ def init_app(self, app: Any, secret_key: str | List[str], salt: str) -> None:
         self.serializer = URLSafeSerializer(secret_key)
         self.salt = salt
 
-    def sign(self, to_sign: str | NotificationDictToSign) -> str | bytes:
```

Comment:
Uncoupling the notification model from the `CryptoSigner`. It will only have a `str` to sign when that is called.